app/src/main/java/org/fossasia/openevent/general/search/SearchTypeViewModel.kt

```diff
@@ -0,0 +1,32 @@
+package org.fossasia.openevent.general.search
+
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+import io.reactivex.android.schedulers.AndroidSchedulers
+import io.reactivex.disposables.CompositeDisposable
+import io.reactivex.schedulers.Schedulers
+import org.fossasia.openevent.general.event.EventService
+import org.fossasia.openevent.general.event.types.EventType
+import timber.log.Timber
+
+class SearchTypeViewModel(
+    private val eventService: EventService
+) : ViewModel() {
+
+    private val compositeDisposable = CompositeDisposable()
```

Comment:
You're not disposing it

Reply:
@iamareebjamal how to dispose it a little bit of help there

Reply:
See other ViewModels

Reply:
already done please review